### PR TITLE
UI completion

### PR DESCRIPTION
**Pull request by pgrenaud**
Should be over now.

### CODE REVIEW

**Comment by S0yKaf** [comment]
:+1:  :420:

File: public/partials/contact.html [hunk 1]
```diff
@@ -2,5 +2,16 @@
   <div class="page-header">
     <h1>Nous joindre</h1>
   </div>
+
+  <p>Vous pouvez nous joindre, pour toute question, commentaire ou demande, par courriel à l'adresse email suivante: <strong>&#105;&#110;&#102;&#111;&#64;&#108;&#97;&#110;&#109;&#111;&#109;&#111;&#46;&#99;&#97;</strong></p>
```
Comment (jdupl) [inline public/partials/contact.html:6]: gj

File: public/partials/terms.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    <li>Toute forme de tricherie ou de conduite antisportive ne sera pas toléré</li>
+    <li>Toute utilisation abusive de la connexion internet sera sanctionné</li>
+    <li>L'échange illégal de matériel et de logiciel protégé par la loi portant sur la propriété intellectuelle est strictement interdite</li>
+    <li>La consommation et la possession de drogue ou d’alcool est strictement interdite</li>
+    <li>Le visionnement, l’échange et la vente de matériel explicite ou pornographique est strictement interdite</li>
+  </ul>
+
+  <h2>Clause de non-responsabilité</h2>
+  <p>L'ACCÈS AU LAN MONTMORENCY EST FOURNIE "TELLE QUELLE", SANS GARANTIE D'AUCUNE SORTE, EXPLICITE OU IMPLICITE, NOTAMMENT SANS GARANTIE DE QUALITÉ, DE FONCTIONNEMENT OU D'AMUSEMENT. EN AUCUN CAS, LES AUTEURS DU LAN MONTMORENCY OU LE COLLÈGE MONTMORENCY NE SERONT RESPONSABLES DE TOUT DOMMAGE, RÉCLAMATION OU AUTRE RESPONSABILITÉ, QUE CE SOIT DANS LE CADRE DE LA PARTICIPATION, D'UN DÉLIT OU AUTRE, EN PROVENANCE DE, CONSÉCUTIF À OU EN RELATION AVEC L'ACCÈS AU LAN MONTMORENCY OU SA PARTICIPATION, OU AVEC D'AUTRES ÉLÉMENTS DU LAN MONTMORENCY.</p>
```
Comment (jdupl) [inline public/partials/terms.html:36]: all caps sweg

**Comment by jdupl** [comment]
:100:

**Comment by jdupl** [comment]
closes { #93, #60, #43}

**Comment by pgrenaud** [comment]
Fixes #74 and #79.

**Pull request merged**